packages/evo-marko/src/tags/evo-accordion/index.marko

```diff
@@ -0,0 +1,67 @@
+import type { Input as DetailsInput } from "<evo-details>";
+export interface Input extends Omit<Marko.HTML.UL, `on${string}`> {
+    size?: "regular" | "large";
+    autoCollapse?: boolean;
+    a11yText?: string;
+    details?: Marko.AttrTag<
+        Omit<DetailsInput, "size" | "alignment" | "open" | "openChange">
+    >;
+    open?: number | number[];
+    openChange?: (open: number | number[]) => void;
+}
+
+<const/{
+    class: inputClass,
+    size,
+    details,
+    autoCollapse = false,
+    a11yText,
+    open: inputOpen,
+    openChange,
+    ...htmlInput
+}=input>
+<const/items=[...(details || [])]>
+
+<let/open:=input.open>
+
+<ul
+    class=["accordion", size === "large" && "accordion--large", inputClass]
+    aria-roledescription=(a11yText ?? "accordion")
+    ...htmlInput>
+    <for|item, index| of=items>
+        <li>
+            <evo-details
+                ...item
+                open=(
+                    Array.isArray(open) ? open.includes(index) : open === index
+                )
+                openChange(isOpen) {
+                    if (autoCollapse) {
+                        if (isOpen) {
+                            open = index;
+                        } else {
+                            open = -1;
+                        }
+                    } else {
+                        const openArray = Array.isArray(open)
+                            ? open
+                            : typeof open === "number" && open >= 0
+                              ? [open]
+                              : [];
+
+                        if (!isOpen) {
+                            open = openArray.filter((n) => n !== index);
+                        } else if (!openArray.includes(index)) {
+                            open = [...openArray, index];
+                        }
+                    }
+
+                    if (open !== undefined) {
+                        input.openChange?.(open);
+                    }
+                }>
+                <${item.content}/>
```

Comment:
This is also not needed, if you spread in `...item` then `item.content` is also spread in. Double check, but removing this should be safe to do!